doc/source/tutorial/stats/continuous_relativistic_bw.rst

```diff
@@ -0,0 +1,32 @@
+
+.. _continuous-relativistic_bw:
+
+Relativistic Breit-Wigner Distribution
+======================================
+
+There is a single shape parameter :math:`\rho`. The support is
+:math:`0 \leq x < \infty`
```

Comment:
If we add the PDF and CDF to the tutorial, I'd suggest adding the moments.

Reply:
Agreed